synapse/lib/stormtypes.py

```diff
@@ -1167,6 +1168,50 @@ def __init__(self, valu, path=None):
     async def _methTextStr(self):
         return self.valu
 
+class LibStats(Lib):
+
+    def addLibFuncs(self):
+        self.locls.update({
+            'tally': self.tally,
```

Comment:
Why not just call this ``counter`` ? That is a more generic name than ``tally``.

Reply:
It'll eventually have some other tracking data ( like counting the number of increments to a given value ) so it can produce additional stats like mean(name). Counter seemed a bit too specific.  Open to other names, but wanted something that didn't seem odd when it eventually had some method like mean() stddist() min() max() etc.